contracts/SSVNetwork.sol

```diff
@@ -724,6 +727,18 @@ contract SSVNetwork is OwnableUpgradeable, ISSVNetwork {
         return clusterId;
     }
 
+    function getPod(uint64[] memory operatorIds) external view override returns(bytes32) {
+        _validateOperatorIds(operatorIds);
+
+        bytes32 clusterId = keccak256(abi.encodePacked(operatorIds));
+
+        if (_pods[keccak256(abi.encodePacked(msg.sender, clusterId))].usage.block == 0) {
+            revert PodNotExists();
+        }
+
+        return clusterId;
```

Comment:
How does this returns `clusterId` if method's name is `getPod`?
Shouldn't this method be called `getCluserId(operatorIds)`?
Because it really returns nothing related to pod

Reply:
We will review usage of pods vs cluster in the upcoming versions